simple_bebop_mcp.py

```
@@ -1,48 +1,380 @@
+                    else:  # C4 이하는 화성으로 간주
+                        chord_track.extend(notes)
+            
+            return {
+                'melody': sorted(melody_track, key=lambda x: x['time']),
+                'chords': sorted(chord_track, key=lambda x: x['time']),
+                'filename': os.path.basename(midi_path)
+            }
+        
+        except Exception as e:
+            debug_log(f"MIDI 파일 분석 오류 {midi_path}: {e}")
+            return None
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)